UnityProject/Assets/Scripts/Electricity/Engine/ReactorGraphiteChamber.cs

```diff
@@ -276,10 +276,19 @@ public void PowerOutput()
 				ReactorPipe.pipeData.mixAndVolume.Mix.InternalEnergy + ExtraEnergyGained;
 		}
 
+		try
+		{
+			CurrentPressure = checked((decimal)((ReactorPipe.pipeData.mixAndVolume.Mix.Temperature - 293.15f) *
+									 ReactorPipe.pipeData.mixAndVolume.Mix.Total));
+		}
+		// An OverflowException is thrown at run time under the following condition:
+		// An arithmetic operation produces a result that is outside the range of the data type returned by the operation
+		catch (OverflowException)
+		{
+			// if result is outside the range of decimal - return
+			return;
+		}
```

Comment:
OverflowException: Arithmetic operation resulted in an overflow.
System.Decimal.op_Explicit (System.Single value) (at <437ba245d8404784b9fbab9b439ac908>:0)
ReactorGraphiteChamber.PowerOutput () (at /github/workspace/UnityProject/Assets/Scripts/Electricity/Engine/ReactorGraphiteChamber.cs:281)
ReactorGraphiteChamber.CycleUpdate () (at /github/workspace/UnityProject/Assets/Scripts/Electricity/Engine/ReactorGraphiteChamber.cs:212)
UpdateManager.ProcessDelayUpdate () (at /github/workspace/UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs:289)
UpdateManager.Update () (at /github/workspace/UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs:269)

Reply:
I think this can be fixed by clamping the Mix.Temperature and Mix.Total

Reply:
Is it possible to exceed such a high pressure value in the game (without any bug)? 

```cs
public decimal MaxPressure = 120000;
public const Decimal MaxValue = 79228162514264337593543950335M;
```

![image](https://user-images.githubusercontent.com/42096831/88427949-40237000-cdf4-11ea-8c78-f603cb2c5aad.png)
*To cause OverflowException CurrentPressure must be more than 660234687618869479946199,586125 times bigger than MaxPressure

Reply:
How about checking the maximum and minimum possible values for Mix.Temperature and Mix.Total?

Reply:
I think probably stuff like temperature/total  goes to nan  because there is zero contents inside of the container